hedera-mirror-rest/Dockerfile

```
@@ -1,5 +1,6 @@
-FROM node:18.12.1-alpine3.16
+FROM node:18.12.1-bullseye-slim
 LABEL maintainer="mirrornode@hedera.com"
+ARG DEBIAN_FRONTEND=noninteractive
```

Doesn't need to be a build arg.
```suggestion
ENV DEBIAN_FRONTEND=noninteractive
```